ford/templates/index.html

```diff
@@ -84,54 +84,54 @@ <h4>{{ author }}</h4>
         <div class="col-xs-6 col-sm-{{ width }}">
               <h3>Source Files</h3>
               <ul>
-                {% for src in project.allfiles|sort(attribute='name') %}
```

Comment:
Could the value of 10 be something set in the config file? If so it would be nice to be able to set this count to 0 to entirely suppress this "sneak peek" at the bottom of the main page.

Reply:
I like this idea. Something to set in the project file. We could keep the current behavior that way, and then it could be overridden.